packages/project-access/src/project/cap.ts

```diff
@@ -151,8 +151,9 @@ function findServiceByUri(
     services: { name: string; urlPath: string }[],
     uri: string
 ): { name: string; urlPath: string } | undefined {
-    const searchUri = uri.replace(/[\\/]/g, '/'); // replace all \ and / in service uri due to issues on Windows with backslashes
-    return services.find((srv: { name: string; urlPath: string }) => srv.urlPath.replace(/[\\/]/g, '/') === searchUri);
+    const uniformUri = (uri: string) => uri.replace(/[\\/]/g, '/').replace(/(?:^\/)|(?:\/$)/g, '');
```

Comment:
IMO, the original comment should be kept and extended to explain the 2nd replace is used to remove leading and trailing slashes.

Reply:
changed it and added comment